docs/_includes/partials/component/sidenav.njk

```diff
@@ -2,14 +2,14 @@
   <ul>
     <li>
       <uxdot-sidenav-item {{ 'active' if page.url ===  '/'}}>
-        <a href="{{ '/' | url }}">Home</a>
```

Comment:
Most likely is a good idea to go back and remove all these.  I should have done so prior to the docs PR merge.  My thought at the time was about the `/v2/` etc directories and the `pathPrefix`.  Was going to try it and the `<base>` plugin and it became an incomplete thought.  

> If you need to deploy your site to a subfolder
https://www.11ty.dev/docs/filters/url/

I'll split this out into an separate issue and take care of all the removals.

Reply:
https://github.com/RedHat-UX/red-hat-design-system/issues/1677